diff --git a/build.sbt b/build.sbt
index 3123c4b8..487fc428 100644
--- a/build.sbt
+++ b/build.sbt
@@ -184,7 +184,7 @@ lazy val testchipipLib = "edu.berkeley.cs" %% "testchipip" % "1.0-020719-SNAPSHO
 lazy val chipyard = (project in file("generators/chipyard"))
   .sourceDependency(testchipip, testchipipLib)
   .dependsOn(rocketchip, boom, hwacha, sifive_blocks, sifive_cache, iocell,
-    sha3, // On separate line to allow for cleaner tutorial-setup patches
+//    sha3, // On separate line to allow for cleaner tutorial-setup patches
     dsptools, `rocket-dsptools`,
     gemmini, icenet, tracegen, cva6, nvdla, sodor)
   .settings(libraryDependencies ++= rocketLibDeps.value)
@@ -223,11 +223,11 @@ lazy val sodor = (project in file("generators/riscv-sodor"))
   .settings(libraryDependencies ++= rocketLibDeps.value)
   .settings(commonSettings)
 
-lazy val sha3 = (project in file("generators/sha3"))
-  .dependsOn(rocketchip, chisel_testers, midasTargetUtils)
-  .settings(libraryDependencies ++= rocketLibDeps.value)
-  .settings(libraryDependencies ++= chiselTestersLibDeps.value)
-  .settings(commonSettings)
+//lazy val sha3 = (project in file("generators/sha3"))
+//  .dependsOn(rocketchip, chisel_testers, midasTargetUtils)
+//  .settings(libraryDependencies ++= rocketLibDeps.value)
+//  .settings(libraryDependencies ++= chiselTestersLibDeps.value)
+//  .settings(commonSettings)
 
 lazy val gemmini = (project in file("generators/gemmini"))
   .sourceDependency(testchipip, testchipipLib)
