diff --git a/generators/chipyard/src/main/scala/config/RocketConfigs.scala b/generators/chipyard/src/main/scala/config/RocketConfigs.scala
index 197ef49..5e1aa1d 100644
--- a/generators/chipyard/src/main/scala/config/RocketConfigs.scala
+++ b/generators/chipyard/src/main/scala/config/RocketConfigs.scala
@@ -328,7 +328,7 @@ class Sha3RocketConfig extends Config(
   new chipyard.config.WithUART ++
   new chipyard.config.WithNoSPIFlash ++
   new chipyard.config.WithL2TLBs(1024) ++
-  new sha3.WithSha3Accel ++                                // add SHA3 rocc accelerator
+//  new sha3.WithSha3Accel ++                                // add SHA3 rocc accelerator
   new freechips.rocketchip.subsystem.WithNoMMIOPort ++
   new freechips.rocketchip.subsystem.WithNoSlavePort ++
   new freechips.rocketchip.subsystem.WithInclusiveCache ++
